examples/documents/example.proto

```diff
@@ -35,6 +35,78 @@ message AllFieldTypesSalts {
   bytes time_stamp_value = 2;
 }
 
+// Placeholder for Nested + Repeated Fields
+message SimpleItem {
+  string valueA = 1;
+}
+
+message SaltedSimpleItem {
+  bytes valueA = 1;
+}
+
+message RepeatedItem {
+  repeated SimpleItem valueA = 1;
+  string valueB = 2;
+}
+
+message SaltedRepeatedItem {
+  repeated SaltedSimpleItem valueA = 1;
+  bytes valueALength = 2;
+  bytes valueB = 3;
+}
+
+message TwoLevelRepeatedDocument {
+  string valueA = 1;
+  repeated RepeatedItem valueB = 2;
+}
+
+message SaltedTwoLevelRepeatedDocument {
+  bytes valueA = 1;
+  repeated SaltedRepeatedItem valueB = 2;
+  bytes valueBLength = 3;
+}
+
+message SimpleRepeatedDocument {
+  string valueA = 1;
+  string valueB = 2;
+  repeated string valueC = 3;
+}
+
+message SaltedSimpleRepeatedDocument {
+  bytes valueA = 1;
+  bytes valueB = 2;
+  repeated bytes valueC = 3;
+  bytes valueCLength = 4;
```

Comment:
Can we add more documentation about how nested fields are flattened?

Reply:
I will include a section in the README.